holoviews/util/command.py

```diff
@@ -1,16 +1,28 @@
+#! /usr/bin/env python
```

Comment:
Is that the recommended most cross-platform approach?  Presumably that won't work on Windows, but just "python" wouldn't work alone?